package/tests/test_api/test_rest/test_responses.py

```diff
@@ -200,6 +200,7 @@ def assert_example_data(response_data):
             "children": [
                 {"id": "0ecea0de", "type": "data"},
                 {"id": "f1f1425b", "type": "parameters"},
+                {"id": "f0ebef01", "type": "parameters"},
```

Comment:
So was part of the problem that the `expected_modular_pipelines` in this test wasn't actually correct?